docs/docs/policy-reference/_examples/graphs/reachable/config.json

```diff
@@ -1,6 +1,8 @@
 {
-	"showInput": false,
-	"showData":  false,
-	"showTitles": false,
-	"command": "data.graph_reachable_example.result"
+  "showInput": false,
+  "showData": false,
+  "showTitles": false,
+  "titleSize": 5,
+  "collapsible": true,
```

Comment:
I am unsure that I see any examples using this? I can't seem to see any detail/summary tags on the page even when this is true.

What's the main reason to have this option? I feel like we might be able to get away without this since our next move is to migrate this content to the category pages?

Reply:
`--ifm-background-surface-color` perhaps https://docusaurus.community/knowledge/design/css/variables/ ?

Reply:
> Do you think it's likely we're going to need this on the category pages though - my feeling is that we might actually not need this since those pages will be shorter?

Probably not unless people browse the docs on phone or vertical monitors a lot

Reply:
I think if you're not planning to use it on the category pages we might be able to thin down this PR without this function.